### DISCUSSION TITLE
When will --now-playing be a thing?

### DISCUSSION THREAD

**Discussion by ElitoGame** [rocksdanister/lively#1325] [General]
Hello!
I am interested if the --now-playing feature is going to be released sometime? I've been using Lively for 2-3 years now and I enjoy working on my web wallpapers. However, I've been kinda stuck at getting the --now-playing to work. It's been part of the Wiki for quite a long time now and was also announced together with the system data, which works like a charm, yet it's still in the NOT READY state.

Would be great to hear if this update is going to be released, or if it's too insignificant compared to the Gallery and other planned features. If there are any major issues with it, what would be the status of those then?
Thanks for your time, awesome program btw! =D

**Comment by rocksdanister** [comment]
>I've been using Lively for 2-3 years now and I enjoy working on my web wallpapers.

Hi great to find a long time Lively fan 😀

I have not looked into the Lively web API for sometime because nobody except a few were making (or sharing to be exact) web wallpapers and giving feedback on it.

>Would be great to hear if this update is going to be released, or if it's too insignificant compared to the Gallery and other planned features.

Since Lively has no analytics at the moment it is hard for me to know which features are popular, based on the emails I received Gallery seemed important so I worked on that.

>If there are any major issues with it, what would be the status of those then?

Looks like I missed creating an issue for [Now playing](https://github.com/rocksdanister/lively/wiki/Web-Guide-V-:-System-Data#--system-nowplaying), can you make one?

**Reply by ElitoGame** [reply]
Sure thing =)
Thanks for the quick reply! Should've posted in here earlier apparently haha!

**Comment by roxxel** [comment]
Hello, it was implemented now on our gallery development repo, it will be a thing after public gallery release
